conda/environments/builddocs_py37.yml

```diff
@@ -0,0 +1,31 @@
+name: rapids_dev
+channels:
+- nvidia
+- conda-forge
+- rapidsai
+dependencies:
+- cudatoolkit=10.0
+- clangdev
+- cmake>=3.12
+- python>=3.7,<3.8
+- numba>=0.40
+- pandas>=0.23.4
+- pyarrow=0.12.0
+- boost
+- cffi>=1.10.0   
```

Comment:
is this required for cudf? I think you might be able to remove it

Reply:
I just copied this big blob of dependencies from @randerzander 's conda env to be honest...

Reply:
I just tried doing a "conda uninstall cffi" and it looks like it's going to gut my entire env as a result, so I think it is needed

Reply:
interesting... I wonder why